tests/integration/start_hello.bats

```diff
@@ -62,3 +62,18 @@ function teardown() {
   [ "$status" -eq 0 ]
   [[ ${lines[0]} =~ [0-9]+ ]]
 }
+
+
+@test "runc run umask" {
+  update_config '(.. | select(. == "umask")) |= 0077'
+
+  # run hello-world
+  runc run test_hello
```

Comment:
This will run to completion so your runc exec will fail.

Reply:
I suggest modeling the test after https://github.com/opencontainers/runc/blob/master/tests/integration/mask.bats

Reply:
Should it live in that file then? I was also unsure which file to put the test in